docs/proposals/backup-restore-multi.md

```diff
@@ -0,0 +1,881 @@
+✓ Pulled application/vnd.docker.container.image.v1+json                     8.14/8.14 KB 100.00%  12ms
+✓ Pulled application/vnd.docker.image.rootfs.diff.tar.gzip                  9.56/9.56 MB 100.00%   18s
+Pulled tag 1.25-alpine with 0 referrer(s)
+
+Exporting to base-images.tar
+Exported to base-images.tar (42.8 MB)
+Successfully backed up 3 tag(s) from 3 repository(ies) in 52s.
+```
+
+#### Example 2: Backup Specific Tags from Multiple Repositories
+
+Backup specific version tags from different base images:
+
+```bash
+oras backup --output versioned-bases.tar \
+  --include-referrers \
+  docker.io/library/alpine:3.18.0,3.19.0 \
+  docker.io/library/busybox:1.35.0,1.36.1 \
+  docker.io/library/redis:7.2-alpine
+```
+
+**Output:**
+```console
+Found 3 repository(ies) with 5 tag(s):
+  - docker.io/library/alpine: 3.18.0, 3.19.0
+  - docker.io/library/busybox: 1.35.0, 1.36.1
+  - docker.io/library/redis: 7.2-alpine
+
+Backing up artifacts...
+✓ Pulled docker.io/library/alpine:3.18.0 with 0 referrer(s)
+✓ Pulled docker.io/library/alpine:3.19.0 with 0 referrer(s)
+✓ Pulled docker.io/library/busybox:1.35.0 with 0 referrer(s)
+✓ Pulled docker.io/library/busybox:1.36.1 with 0 referrer(s)
+✓ Pulled docker.io/library/redis:7.2-alpine with 0 referrer(s)
+
+Exporting to versioned-bases.tar
+Exported to versioned-bases.tar (98.5 MB)
+Successfully backed up 5 tag(s) from 3 repository(ies) in 1m 15s.
+```
+
+## Multi-Repository Restore
+
+### Basic Syntax
+
+Multi-repository restore supports several modes to accommodate different use cases:
+
+**Mode 1: Direct Restore (Same Structure)**
+```bash
+oras restore [flags] --input <path> <target-registry>
+```
+
+**Mode 2: Selective Restore**
+```bash
+oras restore [flags] --input <path> <target-repo1> <target-repo2> [<target-repo3>...]
+```
+
+**Mode 3: Restore with Rules**
+```bash
+oras restore [flags] --input <path> --rules <rules-file>
+```
+
+### Behavior
+
+- **All repositories**: If only target registry specified, all repositories from backup are restored to that registry
+- **Selective repositories**: Specific repository destinations can be provided
+- **Repository mapping**: Complex mappings via rules file
+- **Name preservation**: By default, repository and tag names from backup are preserved
+- **Flexible targets**: Can restore to different registries than original backup source
+
+### Restore Modes
+
+#### Mode 1: Direct Restore to Target Registry
+
+Restore all backed-up repositories to a target registry, preserving original repository paths:
+
+```bash
+oras restore --input base-images.tar registry.mycompany.com
+```
+
+**Behavior:**
+- `docker.io/library/busybox` → `registry.mycompany.com/library/busybox`
+- `docker.io/library/alpine` → `registry.mycompany.com/library/alpine`
+- `docker.io/library/nginx` → `registry.mycompany.com/library/nginx`
+
+**Output:**
+```console
+Loaded backup archive: base-images.tar (42.8 MB)
+Found 3 repository(ies) with 3 tag(s):
```

Comment:
Printing out each tag could be a challenge to read if a repository has a large number of tags

Reply:
Deferred to #1948